app/models/miq_ae_yaml_import.rb

```diff
@@ -222,13 +222,29 @@ def process_method(class_obj, ruby_method_file_name, method_yaml)
     if method_attributes['location'] == 'inline'
       data = load_method_ruby(ruby_method_file_name)
       method_yaml.store_path('object', 'attributes', 'data', data) if data
+    elsif method_attributes['location'] == 'playbook'
+      convert_playbook_attributes(method_attributes['options'], ruby_method_file_name)
     end
     method_obj = MiqAeMethod.find_by(:name => method_attributes['name'], :class_id => class_obj.id) unless class_obj.nil?
     track_stats('method', method_obj)
     method_obj ||= add_method(class_obj, method_yaml) unless @preview
     method_obj
   end
 
+  def convert_playbook_attributes(options, ruby_method_file_name)
+    ae_manager = ManageIQ::Providers::EmbeddedAnsible::AutomationManager
+    begin
+      options[:repository_id]       = AnsibleRepositoryController.model.find_by(:name => options[:repository_name]).id.to_s if options[:repository_name]
+      options[:playbook_id]         = ae_manager::Playbook.find_by(:name => options[:playbook_name]).id.to_s                if options[:playbook_name]
```

Comment:
@pkomanek During import how do we tell the user that the repository could not be found of playbook_name could not be found, the error message has to to tell you what exactly is missing